dbt_project.yml

```diff
@@ -0,0 +1,38 @@
+
+# Name your project! Project names should contain only lowercase characters
+# and underscores. A good package name should reflect your organization's
+# name or the intended use of these models
+name: 'trello_API'
```

Comment:
In the `dbt_project.yml` could you please change the name to 'tap_trello`

Reply:
okay

Reply:
resolved!